codes/1946_py3_sjskgks.py

```diff
@@ -0,0 +1,16 @@
+import sys
+input = sys.stdin.readline
+
+for case in range(int(input())):
+    numOfapp = int(input())
+    applicants = [list(map(int, input().split())) for _ in range(numOfapp)]
+    applicants.sort()

```

Comment:
큰 상관은 없는데 sorted를 쓰면 한줄에 쓸 수 있습니다.
```suggestion
    applicants = sorted([list(map(int, input().split())) for _ in range(numOfapp)])

```